src/java.base/linux/native/libnet/linux_close.c

```diff
@@ -40,6 +40,13 @@
 #include "jvm.h"
 #include "net_util.h"
 
+#if (__STDC_VERSION__ >= 199901L)
+#include <stdbool.h>
+#else
+#define true 1
+#define false 0
+#endif
+
```

Comment:
Since the file already includes `jvm.h`, you can use `JNI_TRUE`/`JNI_FALSE` to avoid introducing more ifdefs

Reply:
Andrei, thanks. Pushed.